cypress/integration/auth.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('Authentication', () => {
+    it('log in with invalid credentials', () => {
+      cy.visit('/')
+      cy.get('input[name="username"]').type('invalid')
+      cy.get('input[name="password"]').type('invalid')
+      cy.get('#kc-login').click()
+      cy.get('.kc-feedback-text').should('have.text', 'Invalid username or password.')
+    })
+
+    it('log in with valid credentials', () => {
+      cy.visit('/')
+      cy.get('input[name="username"]').clear()
+          .type('ona-admin')
+      cy.get('input[name="password"]').type('Amani123')
```

Comment:
Risky to have this exposed in the public repo.  Lets find another way to load this secret here.

Reply:
We should also probably consider this password compromised and change it.  cc: @dubdabasoduba @KipSigei

Reply:
Done